### DISCUSSION TITLE
Link de formulario de Vuex

### DISCUSSION THREAD

**Discussion by AliciaGaona** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#537] [General]
Hola compañeros y compañeras, espero se encuentren bien, alguien podría informarme que paso con el link para entregar la practica de Vuex, según yo teníamos hasta mañana 27 de Marzo, pero no encuentro el link :(.

Espero tengan un hermoso fin de semana.

Saludos.

**Comment by OscarPedroza** [comment]
Creo que aun no esta el link del formulario de entrega compañero, solo el de la práctica, tal vez hoy o mañana se publique.

**Comment by HugoHerreraR** [comment]
aun no se ha publicado el enlace para la entrega
